prestoadmin/server.py

```diff
@@ -231,17 +230,18 @@ def service(control=None):
 
 
 def check_status_for_control_commands():
-    client = PrestoClient(env.host, env.user)
+
     print('Waiting to make sure we can connect to the Presto server on %s, '
           'please wait. This check will time out after %d minutes if the '
           'server does not respond.'
           % (env.host, (RETRY_TIMEOUT / 60)))
-    if check_server_status(client):
+    if check_server_status():
         print('Server started successfully on: ' + env.host)
     else:
-        error('Server failed to start on: ' + env.host +
-              '\nPlease check ' + lookup_server_log_file(env.host) + ' and ' +
-              lookup_launcher_log_file(env.host))
+        warn('Could not verify server status for: ' + env.host +
```

Comment:
changed to warn since not being able to verify doesn't indicate something is wrong (could be something was wrong with the coordinator, e.g.).